modules/mixins/modelMixin.js

```diff
@@ -0,0 +1,34 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', '../registry'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('../registry'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.Mixins = root.Torso.Mixins || {};
+    root.Torso.Mixins.model = factory(root._, root.Torso.registry);
+  }
+}(this, function(_, registry) {
+  'use strict';
+  /**
+   * The base for a model
+   * @module Torso
+   * @namespace Torso.Mixins
+   * @class  modelMixin
+   * @author kent.willis@vecna.com
+   */
+  return {
+    __postInitialize: function() {
+      registry.modelInitialized(this);
+    },
+
+    dispose: function() {
```

Comment:
should we be worried about this keeping references to models that would have otherwise been un-referenced and therefore garbage collected?